corehq/apps/users/views/mobile/users.py

```diff
@@ -1583,6 +1584,19 @@ def post(self, request, *args, **kwargs):
                             change_messages=UserChangeMessage.mobile_account_confirmed_for_domain(self.domain))
             if hasattr(self, 'send_success_sms'):
                 self.send_success_sms()
+            else:
+                context = {
+                    "new_mobile_worker": self.user.raw_username,
```

Comment:
`self.user` is memoized, and I believe that's cached for the life of the instance.  I wonder if doing `self.user` here would potentially get a stale user (without the values updated above). If so, then we should use `user.raw_username` here and below (same with email)

Reply:
ooh yes you're right, idk why I added the `self.` back in there when the success banner right above it uses `user.raw_username`